src/open_inwoner/accounts/models.py

```diff
@@ -313,6 +313,15 @@ def has_verified_email(self):
             "Indicates if the user wants to receive notifications for updates concerning plans and actions."
         ),
     )
+    previous_login = models.DateTimeField(
+        _("previous login"),
```

Comment:
```suggestion
        _("Previous Login"),
```
(for uniformity; see the other verbose fields)